### PR TITLE
(#1182) Remove some more static call to assert

### PR DESCRIPTION
**Pull request by victornoel**
This is for #1182 

I had to increment cactoos-matchers version to benefit from bugfixes (llorllale/cactoos-matchers#132) that were preventing tests to pass with `Assertion`.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1209](https://github.com/yegor256/cactoos/issues/1209) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1209](https://github.com/yegor256/cactoos/issues/1209) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/d5eb99f6-ec48-40dc-b238-21b2ae388024); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by victornoel** [comment]
@fabriciofx ping

**Comment by paulodamaso** [comment]
@fabriciofx please go on

**Comment by victornoel** [comment]
@paulodamaso can you refuse it for @fabriciofx so that someone else is assigned?

**Comment by paulodamaso** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/cactoos/issues/1209#issuecomment-548769233))

@paulodamaso There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 12@b7e726625068, thread: PQ-C63314D6Z
com.zerocracy.claims.ClaimIn[214] java.lang.IllegalArgumentException: Parameter "age" not found in "Order was canceled" among: [comment, job, issue, repo, cause, _parent_sig, flow, login]
```

[0.50.34](https://github.com/zerocracy/farm): CID: [c6ab51d4-04b2-46b3-b2f9-b1568467eab9](https://www.0crat.com/footprint/C63314D6Z/c6ab51d4-04b2-46b3-b2f9-b1568467eab9), Type: "Order was canceled"

File: src/test/java/org/cactoos/io/ReaderOfTest.java [hunk 1]
```diff
@@ -49,102 +48,101 @@
...
-    }
-
     @Test
     public void readsEmpty() throws Exception {
-        MatcherAssert.assertThat(
-            new TextOf(new ReaderOf("")).asString(),
-            new IsEqual<>("")
-        );
+        new Assertion<>(
+            "must read empty string",
```
Comment (paulodamaso) [inline src/test/java/org/cactoos/io/ReaderOfTest.java]: @victornoel I think that would be better to create a constant with the `""` (empty string), WDYT?

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/cactoos/issues/1209#issuecomment-548769233))

@paulodamaso This pull request [#1209](https://github.com/yegor256/cactoos/issues/1209) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C63314D6Z/054a0b6e-9b14-4035-bf15-ce2ea779c490); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

File: src/test/java/org/cactoos/io/ReaderOfTest.java (hunk 1 above)
Comment (victornoel) [reply src/test/java/org/cactoos/io/ReaderOfTest.java]: @paulodamaso you mean a variable in the test for the empty string? I can do that yes

**Comment by victornoel** [comment]
@paulodamaso thank you. I changed the messages because it seems it is the accepted way of describing an assertion with a positive sentence starting with "must". It has been asked to me many times in previous PRs.

**Comment by paulodamaso** [comment]
@victornoel Could you please change these messages back? As I understand, this message would be as like the error message for the assertion failure. This behavior is consistent with many other EO projects, so I think that we should stick to the standard

**Comment by victornoel** [comment]
@paulodamaso that's my point: I understood the standard in EO projects was to use a positive sentence starting with must.

**Comment by paulodamaso** [comment]
@victornoel It seems that we have both cases in `cactoos`, so we have two standards. There is really no problem on this, since the messages fulfill their purpose of informing the assertion result. Taking this into account you can leave the messages as you reworded them, but please just capitalize the first word in each sentence then.

**Comment by victornoel** [comment]
@paulodamaso sorry for the delay, it is done and rebased on master

**Comment by paulodamaso** [comment]
@victornoel thanks for the fixes

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1209#issuecomment-550321355))

@paulodamaso This is what I know about this job in [C63314D6Z](https://www.0crat.com/p/C63314D6Z), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1209](https://github.com/yegor256/cactoos/issues/1209) is [in scope](http://www.zerocracy.com/policy.html#14) for 25days
  * The role is `REV`
  * The job is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) for [5days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @victornoel[/z](https://www.0crat.com/u/victornoel): This user reported the ticket
    * @fabriciofx[/z](https://www.0crat.com/u/fabriciofx): User was resigned from the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231209%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by paulodamaso** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
> @0crat quality acceptable ([here](https://github.com/yegor256/cactoos/issues/1209#issuecomment-552158993))

@paulodamaso There is no quality review for [#1209](https://github.com/yegor256/cactoos/issues/1209), no performer

**Comment by 0crat** [comment]
Code review was too long (10 days), architects (@paulodamaso) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
The job [#1209](https://github.com/yegor256/cactoos/issues/1209) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Pull request merged**